unittests/DependencyScan/Features.cpp

```diff
@@ -27,7 +27,7 @@ testHasOption(llvm::opt::OptTable &table, options::ID id,
               const std::unordered_set<std::string> &optionSet) {
   if (table.getOption(id).hasFlag(swift::options::FrontendOption)) {
     auto name = table.getOptionName(id);
-    if (strlen(name) > 0) {
+    if (!name.empty() && name[0] != '\0') {
```

Comment:
This is just due to the option changes again, `!name.empty()` here is fine IMO. I know a `StringRef` can contain `\0` but it's not going to happen here.